digitaltwin_client/samples/digitaltwin_sample_environmental_sensor/digitaltwin_sample_environmental_sensor.c

```diff
@@ -524,6 +524,11 @@ static void DigitalTwinSampleEnvironmentalSensor_ProcessPropertyUpdate(const DIG
     {
         DigitalTwinSampleEnvironmentalSensor_BrightnessCallback(dtClientPropertyUpdate, propertyCallbackContext);
     }
+    else if (strcmp(dtClientPropertyUpdate->propertyName, digitaltwinSample_DeviceStateProperty) == 0)
+    {
+        LogInfo("ENVIRONMENTAL_SENSOR_INTERFACE: Property name <%s>, last reported value=<%.*s>", 
+            dtClientPropertyUpdate->propertyName, (int)dtClientPropertyUpdate->propertyReportedLen, dtClientPropertyUpdate->propertyReported);
```

Comment:
>     [](start = 8, length = 4)

Nit - please make the dtClientPropertyUpdate line up under the " of the LogInfo line.

Reply:
This is one statement broken into 2 lines...  We are only doing LogInfo in the else if case.